h/subscribers.py

```diff
@@ -3,11 +3,8 @@
 
 def add_renderer_globals(event):
     request = event['request']
-    # Set the base url to use in the <base> tag
-    event['base_url'] = request.resource_url(request.root, '')
-    # Set the service url to use for API discovery
-    event['service_url'] = request.resource_url(request.root, 'api', '')
-    # Allow templates to check for feature flags
```

Comment:
They didn't add anything to me, but maybe i know too much?

The whole purpose of this event is to add globals for the renderer and the values themselves are pretty self describing. It didn't seem necessary to explain how the templates use these values. If one wonders, one can go see.

In particular, I didn't feel like it made sense for the comments to suggest only one particular use of each of the routes.